images/win/Windows2016-Azure.json

```diff
@@ -984,6 +989,12 @@
                 "{{ template_dir }}/scripts/Installers/Validate-DiskSpace.ps1"
             ]
         },
+        {
+            "type": "powershell",
+            "inline": [
+                "pwsh -File '{{user `helper_script_folder`}}SoftwareReport\\SoftwareReport.Generator.ps1'"
```

Comment:
```
"helper_script_folder": "C:\\Program Files\\WindowsPowerShell\\Modules\\",
```
I think it is bad location for software report. Can we find better one?

Reply:
The same for Windows 2019

Reply:
Move to `image_folder`